pkgs/development/python-modules/ical/default.nix

```diff
@@ -44,6 +45,7 @@ buildPythonPackage rec {
     pytest-benchmark
     pytest-golden
     pytestCheckHook
+    pyyaml
   ];
 
   # https://github.com/allenporter/ical/issues/136
```

Comment:
Can this test be enabled again? Looks like it has been fixed in the latest release?